hr_attendance_management/models/hr_attendance.py

```diff
@@ -54,7 +53,7 @@ def open_attendance(self):
     @api.depends('check_in')
     def _compute_date(self):
         for attendance in self.filtered('check_in'):
-            attendance.date = attendance.check_in[:10]
+            attendance.date = fields.Date.from_string(attendance.check_in)
```

Comment:
This is wrong, you are assigning a Date object but it should be a string. So it should be fields.Date.to_string(fields.Date.from_string())) which becomes less sexy.

Reply:
I tried, it was worked fine...

Reply:
It can result to unpredicted behaviour especially if timezones changes or so... I think you should change it.